byzcoin/contracts.go

```diff
@@ -206,6 +207,52 @@ func (b BasicContract) VerifyDeferredInstruction(rst ReadOnlyStateTrie, inst Ins
 	return notImpl("VerifyDeferredInstruction")
 }
 
+// AttrInterpreters provides one default attribute verification which check
+// whether the transaction is sent after a certain block index and before
+// another block index.
+func (b BasicContract) AttrInterpreters(rst ReadOnlyStateTrie, inst Instruction) map[string]func(string) error {
```

Comment:
```suggestion
func (b BasicContract) GetAttrInterpreters(rst ReadOnlyStateTrie, inst Instruction) map[string]func(string) error {
```
The function rename would be to avoid confusion with the type itself.

Reply:
I'm changing it to `MakeAttrInterpreters` because it's dynamically creating those interpreters from the given `rst` and `inst`.